src/index.css

```
@@ -1,14 +1,16 @@
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+@import "tw-animate-css";
+@config "../tailwind.config.ts";
+
+@custom-variant dark (&:is(.dark *));
 
 @layer base {
   @font-face {
-    font-family: "Virgil";
+    font-family: "Excalifont";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Excalifont" (font-family-name-quotes)

(font-family-name-quotes)